frappe/core/doctype/doctype/doctype.json

```diff
@@ -488,11 +489,18 @@
    "fieldtype": "Table",
    "label": "Links",
    "options": "DocType Link"
+  },
+  {
+   "default": "0",
+   "fieldname": "allow_merge_with_existing",
+   "fieldtype": "Check",
+   "label": "Allow merge with existing"
```

Comment:
How about having "Allow Merging" as a `label` and add a `description` with "This allows documents of {doctype} to be merged" instead?

Reply:
Allow Merge doesn't really give you the context. Anyway this feature exists even now, and people are familiar with it, so dint change the name of the label or the fieldname